frontend/src/app/components/wp-fast-table/handlers/row/click-handler.ts

```diff
@@ -62,11 +62,6 @@ export class RowClickHandler implements TableEventHandler {
       return true;
     }
 
-    // Ignore links
-    if (target.is('a') || target.parent().is('a')) {
```

Comment:
Why did you remove this? I remember to added this because the link itself has a `uiSref` that should open the split view. And if you click it with a modifier to open in new window or tab, this line will prevent opening the details in the old window/tab

Reply:
Because exactly the same code is already written a few lines before: https://github.com/opf/openproject/blob/bf86eef503a37da30138e6cef0fb0273c75c6cf8/frontend/src/app/components/wp-fast-table/handlers/row/click-handler.ts#L45

Reply:
That answer I can accept :laughing: